modules/visual_script/visual_script_editor.cpp

```diff
@@ -1040,7 +1041,7 @@ void VisualScriptEditor::_member_button(Object *p_item, int p_column, int p_butt
 			} else if (p_button==0) {
 
 				signal_editor->edit(name);
-				edit_signal_dialog->set_title(TTR("Editing Signal:")+" "+name);
+				edit_signal_dialog->set_title(TTR("Editing Signal: " + name));
```

Comment:
Either what @vnen proposed, or `set_title(TTR("Editing Signal:") + " " + name)` if being able to move the substitution marker around is not necessary for translation. As @eska014 mentioned, moving the white-space out of the TTR call is better to ensure that translators won't remove it.